landscaper/common/generate/openapi/generate.go

```diff
@@ -68,10 +68,10 @@ func init() {
 		"the absolute vendor directory to parse dependent packages")
 
 	rootCommand.Flags().StringVar(
-		&rootPackage,
-		"package",
+		&rootPackages,
+		"packages",
 		"",
-		"the root golang package to generate OpenAPI code for. Example: github.com/gardener/gardener/landscaper/pkg/controlplane/apis/imports/v1alpha1")
+		"the root golang packages (colon-seperated) to generate OpenAPI code for. Example: `github.com/gardener/gardener/landscaper/pkg/controlplane/apis/imports/v1alpha1,other-package`")
```

Comment:
Do the backticks contribute to something in this help text? If not, I would suggest to remove them from the help text.